src/legacy/core_plugins/kibana/public/management/sections/index_patterns/index_pattern_table/index_pattern_table.tsx

```diff
@@ -38,13 +39,15 @@ const columns = [
   {
     field: 'title',
     name: 'Pattern',
-    render: (name: string, { id }: { id: string }) => (
-      <EuiButtonEmpty size="xs" href={`#/management/kibana/index_patterns/${id}`}>
+    render: (name: string, props: { id: string; default: boolean }) => (
+      <EuiButtonEmpty size="xs" href={`#/management/kibana/index_patterns/${props.id}`}>
         {name}
+        {props.default && <EuiBadge className="indexPatternList__badge">Default</EuiBadge>}
       </EuiButtonEmpty>
     ),
     dataType: 'string',
-    sortable: true,
+    sortable: (props: { default: boolean; title: string }) =>
+      `${props.default ? '0' : '1'}${props.title}`,
```

Comment:
Can leave a comment describing the UX intention behind this? You could even copy/paste what you put in the PR description. :)